onnxruntime/test/fuzzing/src/test.cpp

```diff
@@ -243,39 +219,58 @@ int processCommandLine(int argc, char* argv[], runtimeOpt& opt)
         Logger::testLog.minLog();
       }
 
-      // create directory for mutated model output 
+      // create directory for mutated model output
       //
-      if(opt.userOpt.writeModel)
-      {
+      if (opt.userOpt.writeModel) {
         opt.mutateModelDirName = L"./mutatemodel";
         std::filesystem::path mutateModelDir{opt.mutateModelDirName};
-        if ( !std::filesystem::exists(mutateModelDir) )
-        {
+        if (!std::filesystem::exists(mutateModelDir)) {
           std::filesystem::create_directory(mutateModelDir);
         }
       }
-    }
-    else
-    {
+    } else {
       printUsage();
       return 2;
     }
   }
 
   return 0;
 }
+static size_t num_ort_exception{0};
+static size_t num_std_exception{0};
+static size_t num_unknown_exception{0};
+static size_t num_successful_runs{0};
+static size_t iteration{0};
+
```

Comment:
Given this is C++ can we wrap this in a class or struct instead of using file scope statics? You can have an instance of this class in main() so it's accessible in the 'catch' statement that tracks these.